src/main/java/io/rpg/config/ConfigLoader.java

```diff
@@ -127,11 +130,16 @@ public Result<GameWorldConfig, Exception> load() {
     if (gameWorldConfigOpt.isEmpty()) {
       return Result.err(new RuntimeException("loadGameWorldConfig returned null config"));
     }
-
     GameWorldConfig gameWorldConfig = gameWorldConfigOpt.get();
 
     logger.info("GameWorldConfig loaded: \n" + gameWorldConfig);
 
+    QuestionPopup.setBackgroundPath(gameWorldConfig.getQuizPopupBackground());
+    TextPopup.setBackgroundPath(gameWorldConfig.getTextPopupBackground());
+    TextImagePopup.setBackgroundPath(gameWorldConfig.getTextImagePopupBackground());
+    TextPopup.setButtonPath(gameWorldConfig.getTextPopupButton());
+    TextImagePopup.setButtonPath(gameWorldConfig.getTextImagePopupButton());
+
```

Comment:
Maybe we should consider moving this code into `Initializer`, as it is not `ConfigLoader` responsibility to initialize any classes. What do you think?